members/views.py

```diff
@@ -443,6 +443,83 @@ def EntryPage(request):
     return render(request, 'members/entry_page.html', {'loginform' : getLogin, 'signupform' : signup})
 
 @xframe_options_exempt
+def volunteerSignup(request):
```

Comment:
Vi beholder bare denne her for nu - men da den er identisk med den for normal signup af børnene, kunne vi nok genbruge koden.